docs/releasehistory.md

```diff
@@ -78,6 +78,7 @@ Releases follow the `major.minor.micro` scheme recommended by [PEP440](https://w
 ### Miscellaneous
 
 - [PR #1822](https://github.com/openforcefield/openff-toolkit/pull/1922): Updates internal tests to use Pytest 8.
+- [PR #2078](https://github.com/openforcefield/openff-toolkit/pull/2078): Updates molecule cookbook molecule.from_qcschema()
```

Comment:
(blocking) This is in the 0.16.8 section, please move it to the `Current Development` section at the top